docs/docs/jupyter-integration.ipynb

```diff
@@ -0,0 +1,306 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "84b4ac4f-6648-4b62-88ed-3df4296baf4c",
+   "metadata": {},
+   "source": [
+    "Our DataFrame is now interactive! We can explore it with filters, sorting, export options, and even run SQL queries against the data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5db657df-c0f0-4dcc-adc5-baf0cebc228a",
+   "metadata": {},
+   "source": [
+    "### Better structure with Tabs\n",
+    "\n",
+    "Notebooks don't have to be a flood of in/out cells that are viewed from top to bottom. We can add depth to our notebooks with tabs!\n",
+    "\n",
+    "Let's create a scatter plot with our dataset from earlier..."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "26253118-525c-403c-ae94-87d777d1422f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = (\n",
+    "    alt.Chart(df)\n",
+    "    .mark_point()\n",
+    "    .encode(x=\"petalLength:Q\", y=\"petalWidth:Q\", color=\"species:N\")\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2c3a6f47-c42e-49a3-8b54-bb5931f7213a",
+   "metadata": {},
+   "source": [
+    "... and bring them both together with a `dp.Select` block!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "85bf2022-ce96-44b4-9de6-f179bed519c9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dp.Select(\n",
+    "    blocks =[\n",
+    "        dp.Plot(fig, label=\"Plot\"),\n",
+    "        dp.DataTable(df, label=\"Data\")\n",
+    "    ]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ec6b989b-d774-4746-9042-87cbbab89df8",
+   "metadata": {},
+   "source": [
+    "Easy! Now we have an interactive plot, and we can switch to that same interactive data explorer from earlier by clicking the \"Data\" tab."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9af41e8a-c7a5-4241-ac01-6cd7382b3833",
+   "metadata": {},
+   "source": [
+    "### Check out the Blocks Catalog\n",
+    "\n",
+    "Enhance your data apps with Datapane's [catalog of blocks](https://docs.datapane.com/catalogs/blocks/), which includes the handy `dp.BigNumber` block."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c1bbee21-3241-41d7-a7e2-d7551a47a5bc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dp.Group(\n",
+    "    dp.BigNumber(\n",
+    "        heading=\"Percentage points\",\n",
+    "        value=\"84%\",\n",
+    "        change=\"2%\",\n",
+    "        is_upward_change=True,\n",
+    "    ),\n",
+    "    dp.BigNumber(\n",
+    "        heading=\"Points\",\n",
+    "        value=\"1234\",\n",
+    "        change=\"200\",\n",
+    "        is_upward_change=False,\n",
+    "    ),\n",
+    "    columns=2,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aac7e2d0-5f5d-41d5-ae6d-dee5628cc872",
+   "metadata": {},
+   "source": [
+    "## Notebook to Data App? One line of code."
```

Comment:
I would structure the tutorial to have this first, i.e.

1. Turn any notebook into an app in 1 line of code
2. Embed interactive components

Reply:
Sure, I'll re-order them.

It was ordered this way so that we're building the app in part 1 (which are embedded too), and then we generate it in part 2. I'll need to add some additional code (e.g. blocks) to the new part 1 so there's something to convert into a notebook.